docker.local/config/0chain.yaml

```diff
@@ -16,26 +14,29 @@ development:
     faucet: true
     zrc20: true
     interest: true
+    fee: false
     miner: true
     multisig: true
     vesting: true
+    magma: false #set true to enable magma
     zcn: true
   txn_generation:
     wallets: 50
+    transactions: 70
+    pour_point: 100
+    pour_amount: 10000
```

Comment:
Are these new added configs being used? Perhaps we should do these changes along with the code that use them.

Reply:
it is not used, removed